logging-modules/flogger/src/main/resources/log4j.xml

```diff
@@ -0,0 +1,99 @@
+<!DOCTYPE log4j:configuration SYSTEM "log4j.dtd" >
```

Comment:
Why we need this file?

Reply:
This file we probably don't need, so I have removed it. I was using it to test that existing configurations actually work, but it's not necessary for the flow of the article